.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 dist
 .DS_Store
 packages/backend/storage/pyramidSaved
+packages/backend/storage/db.sqlite
```

Comment:
i'm using packages/backend/storage for disk storage during development of the backend